src/common/metrics-service/RillIntakeClient.ts

```diff
@@ -15,6 +15,8 @@ export class RillIntakeClient {
   }
 
   public async fireEvent(event: MetricsEvent) {
+    // Debug Telemetry by uncommenting the below line
+    // console.log(event);
```

Comment:
@magorlick You can uncomment this line to see what events are being fired by the application